### ISSUE TITLE
Could not get unknown property kotlin_Version react-native-async-storage_async-storage

### ISSUE THREAD

**Issue by yasinSimma** [react-native-async-storage/async-storage#1041]
### What happened?

get this error after run `npx react-native run-android`
<img width="1049" alt="Screenshot 2023-11-29 at 12 14 51 PM" src="https://github.com/react-native-async-storage/async-storage/assets/115902578/7421dfbb-70d2-422b-813b-646dfd709712">

### Version

1.18.2

### What platforms are you seeing this issue on?

- [X] Android
- [ ] iOS
- [ ] macOS
- [ ] Windows
- [ ] web

### System Information

```true
System:
  OS: macOS 13.5
  CPU: (8) arm64 Apple M1
  Memory: 88.97 MB / 8.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 18.17.0
    path: ~/.nvm/versions/node/v18.17.0/bin/node
  Yarn:
    version: 1.22.19
    path: /opt/homebrew/bin/yarn
  npm:
    version: 9.6.7
    path: ~/.nvm/versions/node/v18.17.0/bin/npm
  Watchman:
    version: 2023.11.06.00
    path: /opt/homebrew/bin/watchman
Managers:
  CocoaPods:
    version: 1.12.1
    path: /opt/homebrew/bin/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 22.4
      - iOS 16.4
      - macOS 13.3
      - tvOS 16.4
      - watchOS 9.4
  Android SDK: Not Found
IDEs:
  Android Studio: 2022.3 AI-223.8836.35.2231.10406996
  Xcode:
    version: 14.3/14E222b
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 11.0.16
    path: /usr/bin/javac
  Ruby:
    version: 2.6.10
    path: /usr/bin/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.4
    wanted: 0.72.4
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Steps to Reproduce

install packages
run android

**Comment by krizzu** [comment]
Yup, it's regression introduced in v1.20.0, please hold on for the fix is here: #1042

**Comment by krizzu** [comment]
v1.21.0 is out, please give it a go 🙏 
https://github.com/react-native-async-storage/async-storage/releases/tag/v1.21.0
